guides/source/action_text_overview.md

```diff
@@ -117,28 +110,60 @@ end
 
 ## Rendering Rich Text Content
 
-By default, Action Text will render rich text content inside an element with the
-`.trix-content` class:
+Instances of `ActionText::RichText` are capable of safely rendering themselves, so embed them directly into the page:
+
+```erb
+<%= @message.content %>
+```
+
+You can learn more about Action Text's sanitization process in the documentation for the [`ActionText::RichText`](https://api.rubyonrails.org/classes/ActionText/RichText.html) class.
```

Comment:
I think we [prefer](https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#wording) to not use `you` in the guides:
```suggestion
Learn more about Action Text's sanitization process in the documentation for the [`ActionText::RichText`](https://api.rubyonrails.org/classes/ActionText/RichText.html) class.
```